pyExample_alg01/pyExample_alg01.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import os
+cur_dir = os.path.dirname(__file__)
+pyd_dir = os.path.join(cur_dir, '../cppExample_alg01/build/Release')
+sys.path.append(os.path.abspath(pyd_dir))
+import rounder
+
+r = rounder.Rounder()
+if len(sys.argv) >= 1:
+    num = sys.argv[1]
```

Comment:
I think using the native python `argparser` would be better here. See [this](https://github.com/EmotiBit/EmotiBit_Biometric_Lib/blob/a7112ed157307df68ddab43fbb5a0a5291c109fd/py/emotibit/tapdetector.py#L453-L464) for example.

I think having a call like `python3 pyExample_alg01.py -i 3.4` where -i suggests `input` improves readability and usage. It also makes this example extensible, if we want to add more arguments like `ROUND_UP` or `ROUND_DOWN` etc.